src/binary/ibc-setup/commands/init.ts

```diff
@@ -46,17 +88,42 @@ export async function run(options: Options) {
       const registryFromRemote = await axios.get(
         'https://raw.githubusercontent.com/confio/ts-relayer/main/demo/registry.yaml'
       );
-      fs.writeFileSync(registryFilePath, registryFromRemote.data, {
-        encoding: 'utf-8',
-      });
+      fs.writeFileSync(registryFilePath, registryFromRemote.data);
     } catch (error) {
       throw new Error(`Cannot fetch ${registryFile} from remote. ${error}`);
     }
   } else if (!fs.lstatSync(registryFilePath).isFile()) {
     throw new Error(`${registryFilePath} must be a file.`);
   }
 
-  const registry = yaml.load(fs.readFileSync(registryFilePath, 'utf-8'));
+  const registry = yaml.load(
+    fs.readFileSync(registryFilePath, 'utf-8')
+  ) as Registry;
   // TODO #75: registry file validation
-  console.log(registry);
+
+  const chainSrc = registry.chains[options.src];
```

Comment:
These could be undefined, right?
We should validate that both `chainSrc` and `chainDest` are objects here, before making the `app.yaml` file. (Try to `--src=foobar` and see what happens.)

Reply:
Yeah, I was going to cover it with registry validation.

- aefe0ec
- ecf1796

Reply:
THe registry is valid, but you don't check if these keys are present, correct?

Seems like 3 lines to add. This is the only blocker before merge

Reply:
It's already there: https://github.com/confio/ts-relayer/pull/83/files#diff-e8c0ad1a88c4869847d33b7631f1c6a11037972cd3ac65ecab3834ef0f0bf617R113